.github/workflows/sync-external-images.yml

```diff
@@ -13,12 +16,13 @@ on:
     paths:
       - "external-images.yaml"
 
-permissions:
-  id-token: write # This is required for requesting the JWT token
-  contents: read # This is required for actions/checkout
+
 
 jobs:
   sync-external-images:
+    permissions:
+      id-token: write # This is required for requesting the JWT token
+      contents: read # This is required for actions/checkout
```

Comment:
I find these comments very useful. Could you please add them everywhere we are overruling the default permissions?

Reply:
done